source/accessible-media/subtitles.html.erb.md

```diff
@@ -15,25 +14,11 @@ contributors:
 support: Developed with support from the <a href="https://www.w3.org/WAI/WCAGTA/">U.S. Access Board, WCAG TA Project</a>
 ---
 
-## Subtitle basics
-
 Subtitles (also known as foreign-language subtitles) are, from a
```

Comment:
“Subtitle” is a weird word. In Germany, the term for captions is “Untertitel für Hörgeschädigte,” literally translated “Subtitles for the hard of hearing”.  I don’t know if there are other forms of subtitles than foreign-language subtitles in English.